docs/projects/amt/reporting-standard/schemas/assessmentcard_schema.json

```diff
@@ -0,0 +1,86 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/MinBZK/ai-validatie/main/docs/projects/amt/reporting-standard/schemas",
+    "title": "Model card schema",
+    "description": "JSON schema for model card",
+    "type": "object",
+    "properties": {
+        "provenance": {
+            "type": "object",
+            "properties":{
+                "git_commit_hash":{
+                    "type": "string",
+                    "description": "Git commit hash of the commit which contains the transformation file used to create this card"
```

Comment:
We could enforce the formats of this and the fields below https://json-schema.org/understanding-json-schema/reference/string#regexp